dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.xml

```diff
@@ -308,6 +308,30 @@
         </if>
         order by instance.start_time desc,instance.id desc
     </select>
+    <select id="countInstanceStateV2" resultType="org.apache.dolphinscheduler.dao.entity.ExecuteStatusCount">
```

Comment:
we should use `org.apache.dolphinscheduler.api.enums.ExecuteType` instead of `org.apache.dolphinscheduler.dao.entity.ExecuteStatusCount`, ExecuteStatusCount if for task status not for workflow

Reply:
resolve